docs/usage-as-end-user.md

```diff
@@ -123,6 +123,9 @@ The `cloudControllerManager.featureGates` contains a map of explicitly enabled o
 For production usage it's not recommend to use this field at all as you can enable alpha features or disable beta/stable features, potentially impacting the cluster stability.
 If you don't want to configure anything for the `cloudControllerManager` simply omit the key in the YAML specification.
 
+The `cloudControllerManager.alphaFeatureGates` contains a map of enabled or disabled feature gates.
+The alpha features can only be enabled in when the `AllAlpha` feature gate is enabled.
```

Comment:
Isn't `AllAlpha` too dangerous? I guess currently there are alpha features that can cause issues when they are enabled. One good example is the `MemoryQoS` feature gate that requires (I expect) cgroup v2 that is currently not supported (disabled) by gardenlinux - https://github.com/gardenlinux/gardenlinux/pull/205.
I guess no one of us would like to dig for issue in a cluster where AllAlpha feature gate is enabled.